### PR TITLE
Remove save button/menu item and warning dialog about unsaved progress

### PR DESCRIPTION
**Pull request by dimitry-**
Bug: http://b/189159722

### CODE REVIEW

**Review by beckerhe — APPROVED** [review]
Nice. Thanks. LGTM

File: src/OrbitQt/orbitmainwindow.cpp [hunk 1]
```diff
@@ -1065,35 +1064,6 @@ void OrbitMainWindow::OnTimerSelectionChanged(const orbit_client_protos::TimerIn
   ui->liveFunctions->OnRowSelected(selected_row);
 }
 
-void OrbitMainWindow::on_actionSave_Capture_triggered() {
-  ShowCaptureOnSaveWarningIfNeeded();
```
Comment (dpallotti) [inline src/OrbitQt/orbitmainwindow.cpp:1069]: I think this is the only use of this method and as such it can be removed too. I might be wrong.

**Review by dpallotti — APPROVED** [review]

File: src/OrbitQt/orbitmainwindow.cpp (hunk 1 above)
Comment (dimitry-) [reply src/OrbitQt/orbitmainwindow.cpp:1069]: right - removed.

**Review by antonrohr — APPROVED** [review]
lgtm

**Pull request merged**